fe/fe-core/src/main/java/org/apache/doris/datasource/HMSExternalCatalog.java

```diff
@@ -162,6 +166,16 @@ public PooledHiveMetaStoreClient getClient() {
     public List<Column> getSchema(String dbName, String tblName) {
         makeSureInitialized();
         List<FieldSchema> schema = getClient().getSchema(dbName, tblName);
+        Optional<ExternalDatabase> db = getDb(dbName);
+        if (db.isPresent()) {
```

Comment:
The above `List<FieldSchema> schema = getClient().getSchema(dbName, tblName);` can be done after this `if` block